os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -132,6 +132,17 @@ extern int g_irq_num;
 #define IS_FAULT_IN_USER_THREAD(fault_tcb)		((void *)fault_tcb->uheap != NULL)
 #define IS_FAULT_IN_USER_SPACE(asserted_location)	(is_kernel_space((void *)asserted_location) == false)
 
+#ifdef CONFIG_SMP
+#define LOG_CPU_INDEX   up_cpu_index()
+#else
+#define LOG_CPU_INDEX   (0)
+#endif
```

Comment:
I don't know why this is here. We can enable / disable the smp. Then, how do other codes call the up_cpu_index?
I think this definition should be in the header which has the prototype of the function.
It looks the `os/arch/arm/include/irq.h`. 
How about moving this there and calling just `up_cpu_index` only here?

And, the function prototype is in `irq.h`. Is it proper? Is it an interrupt content?
@kishore-sn Could you let me know your opinion for the location of prototype?

Reply:
We alredy have this defined in irq.h, no need to redefine it. Instead up_cpu_index can be used directly.
https://github.com/Samsung/TizenRT/blob/98acc0caf1caeafab54d8b869c93858ff73c96ef/os/arch/arm/include/irq.h#L139-L143